api/dogBreeds/dogBreeds-router.js

```diff
@@ -0,0 +1,58 @@
+const express = require(`express`)
+const router = express.Router()
+const Breeds = require(`./dogBreeds-model`)
+const {checkPayload, checkId} = require(`./dogBreeds-middleware`)
+
+router.get(`/`, (req, res) => {
+    Breeds.getAll()
+        .then(breeds => {
+            res.status(200).json(breeds)
+        })
+        .catch(err => {
+            console.log(`Error in dogBreeds router`, err)
+            res.status(400).json({message: `Something went wrong, please try again.`})
+        })
+})
+
+router.get(`/:id`, checkId, (req, res) => {
+    const id = req.params.id
+    Breeds.getById(id)
+        .then(breed => {
+            res.status(200).json(breed)
+        })
+        .catch((err) => {
```

Comment:
Nit: for the routes in the catch block you can keep the console.log but it's also good to return a response of 500 with some message. If an error occurs that goes into the catch the request will hang because nothing is sent back